remote_execution/oss/re_grpc/src/response.rs

```diff
@@ -91,6 +91,9 @@ pub struct TSubsysPerfCount {
 pub struct TActionResult2 {
     pub output_files: Vec<TFile>,
     pub output_directories: Vec<TDirectory2>,
+    // TODO: output_symlinks (use in preference when output_paths mode is used the execution side)
+    // TODO: output_file_symlinks (deprecated)
```

Comment:
Why mark this as a todo? Is it something we should add? Or is this just for when servers return deprecated stuff?